test/docfx.Test/lib/JsonUtilityTest.cs

```diff
@@ -47,7 +47,7 @@ public void TestObjectWithStringProperty(string input)
             var sw = new StringWriter();
             JsonUtility.Serialize(sw, new BasicClass { C = input });
             var json = sw.ToString();
-            var (errors, value) = JsonUtility.DeserializeWithSchemaValidation<BasicClass>(json);
+            var (errors, value) = Deserialize<BasicClass>(json);
```

Comment:
this Deserialize will cause confusion with JsonUtility/YamlUtility,Deserialize, rename it to DeserializeWithSchemaValidation?